### ISSUE TITLE
[5.x]: Only first tab visible when matrix view mode is "as inline editable blocks"

### ISSUE THREAD

**Issue by thupsi** [craftcms/cms#14139]
### Description
Only first tab visible when matrix view mode is "as inline editable blocks"

### Steps to reproduce

1. Create an entry type with a field layout that has at least two tabs.
2. Include this entry type in a matrix field.
3. In the matrix field setting "view mode" select "as inline editable blocks".

### Expected behavior
The inline block should display all the tabs.

### Actual behavior
Only the first tab is displayed.

### Craft CMS version

5.0.0-alpha.5

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
Yeah this is a known limitation of inline block view for now. Multiple tab support (and live condition rules) are on the list.

**Comment by brandonkelly** [comment]
Craft 5.0.0-alpha.7 was just tagged, which adds multiple tab support to inline-editable Matrix blocks :tada:

<img src="https://github.com/craftcms/cms/assets/47792/b643df50-07df-4c39-8fe2-c94efcd17f0e" width="555" height="151" alt="A Matrix block with two tabs">

**Comment by thupsi** [comment]
Thanks @brandonkelly and @i-just, I have already seen that and installed the latest alpha. Looks great! Also, I saw a hint in that PR for something else that should also be a neat Matrix feature but you haven't announced yet 😉
